hasher-matcher-actioner/hmalib/lambdas/bucket_hasher.py

```diff
@@ -0,0 +1,8 @@
+from hmalib.common.logging import get_logger
+
+logger = get_logger(__name__)
+
+
+def lambda_handler(event, context):
+    logger.info("bucket_hasher received event")
```

Comment:
@schatten I don't understand "Would also recommend using the URL pathway because that would reduce IO resource usage." Can you explain more about that?

Reply:
There are presently 3 separate ways to submit an image. 
- upload (encode bytes into base64 as part of request payload) 
- direct - upload (get a signed URL from submission APIs and upload to it)
- url (only provide the URL)

The third one is what I'm recommending because it will reduce the transfer of bytes. In either of the other two, you'll have the following large transfers.
1. Download from `local-s3-bucket` to `bucket-hasher` lambda
2. [if using the 1st mechanism] upload to `api_root` lambda
2. Upload from `bucket-hasher` lambda or `api_root` lambda to `hashing-data` s3 bucket
3. Download from `hashing-data` s3 bucket to `hashing` lambda

When I say IO, I mean using up bandwidth transferring files across the network: Network IO.

Reply:
Yep. Plan is to use `record_content_submission` and `create_presigned_get_url`

Reply:
I'm saying you call the API over http rather than call the python methods. Separation of concerns.